lunches.py

```diff
@@ -390,6 +390,45 @@ def caesar(dom):
 def morgans(dom):
     yield from menicka_parser(dom)
 
+@restaurant("U Mořice", "https://www.menicka.cz/api/iframe/?id=5299", Location.Olomouc)
+def moric(dom):
+    yield from menicka_parser(dom)
+
+def slepico_parser(dom):
+    current_day = datetime.datetime.now().strftime("%-d.%-m.%Y")
+    slepico_regex = re.compile("^(\d\s?[,.]\s?\d\s?l?\s)?(?P<soup>[^+]+)(\s+\+\s+)(\d+g)?(?P<lunch>.*)$")
+    for day_dom in dom.select('.content'):
+        day = day_dom.select_one('h2').text.strip().split(' ', 2)[1]
+        if day != current_day:
+            continue
+
+        parsed_soup = None
+
+        for food in day_dom.select('.soup'):
+            if 'Pro tento den nebylo zadáno menu' in food.text:
+                break
+            matched = re.match(slepico_regex, food.select_one('.food').text)
+
+            if parsed_soup is None:
+                parsed_soup = matched['soup']
+                yield Soup(
+                    parsed_soup
+                )
+
+            yield Lunch(
+                num=0,
```

Comment:
Do you want everything to be zero? If you omit this parameter then it will be numbered automatically